src/ScottPlot/Palette.cs

```diff
@@ -14,6 +14,7 @@ public static class Palette
         public static ScottPlot.Drawing.Palette Aurora => new(new ScottPlot.Drawing.Colorsets.Aurora());
         public static ScottPlot.Drawing.Palette Category10 => new(new ScottPlot.Drawing.Colorsets.Category10());
         public static ScottPlot.Drawing.Palette Category20 => new(new ScottPlot.Drawing.Colorsets.Category20());
+        public static ScottPlot.Drawing.Palette DarkLight => new(new ScottPlot.Drawing.Colorsets.DarkLight());
```

Comment:
`DarkLight` sounds like an oxymoron, maybe `DarkPastel` or `DarkDesaturated` make more sense? It's a minor thing though.

Reply:
I like `DarkPastel`! I'll rename this and merge 👍